### ISSUE TITLE
Unable to save flextable to pdf  using save_as_image

### ISSUE THREAD

**Issue by Pierre-GH** [davidgohel/flextable#522]
Hi David,

I was using the code below to save tables as pdf files.  I have been using it without incident for a few weeks.  Any help would be greatly appreciated.  I have attached a copy of the error message from Adobe when opening the saved files.  Thanks for the assist!

save_as_image(my_ft,path = "my_ft.pdf",zoom = 1,webshot = "webshot2")

![image](https://user-images.githubusercontent.com/127868286/225125099-1334b91f-124a-4304-9c95-9c2181a7ef3b.png)

**Comment by davidgohel** [comment]
yes, there have been lot of complaints about webshot and webshot2, so we decided to stop using it. 

Now `save_as_image()` only use `ragg` and produce png and we are happy with this version. 


If you want PDF, can't you use R Markdown (or Quarto)?

**Comment by Pierre-GH** [comment]
Many thanks for your assistance David.  I am quite new to R and will need to learn how to use R Markdown for that purpose.  I understand this issue has been closed, but I do have one more question.  I was keen on this format largely due to preferences my employers.  I am now using save_as_docx() and functions from your Officer package to achieve the same aim.  The plan is to automate the script so it produces individualized reports for members of the team on a daily or monthly basis depending on data requirements.  Do you know if that is possible using an R Markdown file? Can they be automated in the same was as scripts? I understand it can only work with R Scripts.  Thanks David.

**Comment by davidgohel** [comment]
no problem @Pierre-GH

You can read here what I think about R Markdown / officer choice :) 
https://ardata-fr.github.io/officeverse/index.html#officer
